### PR TITLE
Constitution update, particularly for audit of finances

### PR DESCRIPTION
**Pull request by lwjohnst86**
For #44

### CODE REVIEW

File: Constitution.md [hunk 1]
```diff
@@ -113,11 +116,14 @@ Term of executive positions shall be from May 1st to April 30th of the following
...
 
-The group’s executive or members may not engage in activities that are essentially commercial in nature. This is not intended to preclude the collection of membership fees to cover the expenses of the group, or of charges for specific activities, programs or events, or to prohibit groups from engaging in legitimate fundraising. 
 However, the group will not have as a major activity a function that makes it an on-campus part of a commercial organization, will not provide services and goods at a profit when that profit is used for purposes other than those of the organization, and will not pay salaries to some or all of its officers.
 
+A majority of the executive council will take part in a twice-yearly internal review and audit of the finances and accounting. The goal of such review will be to maintain tidy and organized documentation surrounding the finances, to ensure integrity and accuracy of the financial condition of the organization, and to keep the executive members aware of the financial health of the organization. Audit's shall take place every March and October of every year.
```
Comment (QuLogic) [inline Constitution.md]: Audits

File: Constitution.md [hunk 2]
```diff
@@ -49,31 +51,33 @@ Toronto;
...
 #### The Technical Admin shall:
 
 * Be responsible for the maintenance and upkeep of the [website](uoftcoders.github.io/studyGroup/).
 * Oversee the activity of the [studyGroup repository](https://github.com/UofTCoders/studyGroup) in terms of pull requests
 for new features or weekly lesson material. 
 
-#### Admin
+#### The Admin shall:
 
 * Responsible for internal communications of the council and its members
```
Comment (joelostblom) [inline Constitution.md]: "Be responsible"?

File: Constitution.md [hunk 3]
```diff
@@ -49,31 +51,33 @@ Toronto;
...
 * Inform members of UofT Coders events via email, create event postings for the website and contact instructors to create the event issue; 
 * Record the minutes of the Executive Council meetings;
 * Inform and remind council members of items discussed in the meetings via email;
 * Maintain the master email list for uoftcoders@gmail.com.
 
-#### Special Events Coordinator
+#### The Special Events Coordinator shall:
 
 * Responsible for coordinating and organizing:
```
Comment (joelostblom) [inline Constitution.md]: "Be responsible"?

File: Constitution.md [hunk 4]
```diff
@@ -49,31 +51,33 @@ Toronto;
...
     - Software Carpentry workshops throughout the year (at least one per term)
     - Workshops with various organizations (e.g. CUPE)
     - Any other coding type events that come up (e.g. hackathons, code sprints)
 
-#### RQM Course Coordinator
+#### The RQM Course Coordinator shall:
 
 * Responsible for coordinating and organizing the Reproducible Quantitative Methods (RQM) course (based on [the course by Christie Bahlai](https://cbahlai.github.io/rqm-template/)) at the University of Toronto with the help of the other UofT Coders executive council members;
```
Comment (joelostblom) [inline Constitution.md]: "Be responsible"?

**Pull request merged**